src/app/pages/reports-dashboard/components/reports-global-controls/reports-global-controls.component.scss

```diff
@@ -25,6 +25,38 @@
   }
 }
 
-.reports-dash-global-controls {
+::ng-deep .reports-dash-global-controls {
   gap: 12px;
+
+  ix-slide-toggle {
```

Comment:
1. If there minor stylistic differences, you can ignore them. I'm okay with labels not being in one line.
2. If you still need to use ::ng-deep, it's better to use it after `reports-dash-global-controls` to reduce its scope.